<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find in base</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find string:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="string"/>
   </item>
   <item>
    <widget class="QPushButton" name="find">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pwd_v">
     <property name="text">
      <string>View passwords</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="data">
     <column>
      <property name="text">
       <string>â„–</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Addres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Group</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Login</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Password</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
